src/families/bitcoin/descriptor.test.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import invariant from "invariant";
+import { getCryptoCurrencyById } from "../../currencies";
+import {
+  inferDescriptorFromDeviceInfo,
+  inferDescriptorFromAccount,
+} from "./descriptor";
+import { fromAccountRaw } from "../../account";
+import bitcoinDatasets from "./datasets/bitcoin";
+
+describe("inferDescriptorFromAccount", () => {
```

Comment:
Can you add some tests for:
* wrapped segwit (`49'/0'/x'`)
* testnet accounts (`x'/1'/y'`)

Reply:
duly noted. At the moment however, we don't have segwit account in datasets/bitcoin and i'll wait @Arnaud97234  to provide one for future testing this.
we also don't have testnet datasets

as we try to keep these dataset static and from same QA see we'll add more test a bit later. this was tested on CLI and should work. There is also some chance that more functions will be needed when we will do the LLD integration, so this PR should be good to go as first kickoff of the logic.